transfers/get-accession-number

```diff
@@ -7,9 +7,9 @@ def main(dirname):
     # Expecting a directory name like sometext---dc.identifier---accession
     parts = dirname.rsplit('---', 1)
     try:
-        print('"' + parts[1] + '"')  # Accession ID must be quoted
+        print('"' + parts[1] + '"', end='')  # Accession ID must be quoted
```

Comment:
Thought it looked nicer?  I think I was debugging ast.literal_eval stuff, but this doesn't seem to affect it.